### ISSUE TITLE
[PHP] FQN for aliases

### ISSUE THREAD

**Issue by maarsson** [20Tauri/DoxyDoxygen#172]
Using the setting `"php_generate_fully_qualified_name": true` and also using import aliases it generates docblock with the alias instead of fully qualified class name.
```
    use App\Http\Requests\User\CourseRequest as UserCourseRequest;

    /**
     * { function_description }
     *
     * @param UserCourseRequest $request 
     */
    public function store(UserCourseRequest $request)
    {
    }
```

In this example it should be `@param \App\Http\Requests\User\CourseRequest ( description_of_the_return_value )`

**Comment by 20Tauri** [comment]
Thank you for your report, this case has been corrected and will be included in the next version.

**Comment by 20Tauri** [comment]
Fixed in 1.0.1 (evolution branch only)
